ExpressionEngine/IFunctionDefinition.cs

```diff
@@ -0,0 +1,8 @@
+namespace ExpressionEngine
+{
+    public interface IFunctionDefinition
+    {
+        public string From();
+        public string To();
```

Comment:
we dont need to specificy public on interfaces, its implicit for interfaces. I dont know what is common here, but i seen warnings when adding public myself before. 

I would do string From {get;set;} (see other comment)

Reply:
I agree, see 06f3ee1